packages/peregrine/lib/talons/RootComponents/Category/useCategoryContent.js

```diff
@@ -70,8 +91,75 @@ export const useCategoryContent = props => {
     const categoryDescription = categoryData
         ? categoryData.categories.items[0].description
         : null;
+    const sortingMethods = sortData
+        ? sortData?.products?.sort_fields?.options
```

Comment:
I don't think you need to use optional chaining here, is there any scenario where those fields would be undefined even with returned data? Also if you use the ternary operator on `sortData` you don't need to escape it with optional chaining, since it will always return a value.

Reply:
I had to because the tests were failing. Doing this way made the tests work